### DISCUSSION TITLE
Proxmox VM can't be shutdown

### DISCUSSION THREAD

**Discussion by sshappys** [MichaIng/DietPi#5665] [Software Request] [answered]
大家好
          我是下载了DietPi_Proxmox-x86_64-Bullseye.qcow2这个PVE虚拟机用的系统，安装到是没什么问题，不过感觉有点慢，我只在系统中装了mysql8，内存使用了500多M，一直这个状态，CPU到是没用多少，我的是J1900，6.1.3的虚拟机版本，而且发现无法在PVE上对他进行关机，使用shutdown会异常，只能用stop停止：TASK ERROR: VM quit/powerdown failed - got timeout
不是只出现一次，是一直都是这样，重装一次也是。不知道是不是我的系统性能不够？

**Comment by MichaIng** [comment, marked as answer]
Many thanks for your report.

It would be nice if you could pass your posts here through a translator, like deepL 😉:
> Hello everyone
I downloaded DietPi_Proxmox-x86_64-Bullseye.qcow2, the system used for PVE virtual machine, there is nothing wrong with the installation, but I feel a little slow, I only installed mysql8 in the system, the memory used more than 500M, always this state, the CPU is not used to much, my is J1900, 6.1.3 VM version, and found that I can not shutdown him on PVE, using shutdown will be abnormal, can only use stop stop: TASK ERROR: VM quit/powerdown failed - got timeout
Not just once, but all the time, and even after a reinstallation. I don't know if my system performance is not enough

For ACPI shutdown please try to enable systemd-logind and install qemu-guest-agent:
```sh
systemctl unmask systemd-logind
apt install dbus
systemctl start systemd-logind
apt install qemu-guest-agent
```

**Reply by sshappys** [reply]
OH, sorry ,I will use the translation next time ,thanks.

**Comment by StephanStS** [comment]
@MichaIng : I propose to close this discussion, because our actual Proxmox image got this feature many months ago (see https://dietpi.com/docs/releases/v8_24/#bug-fixes).
